tavern/config_test.go

```diff
@@ -118,7 +118,7 @@ func TestConfigureOAuthFromEnv(t *testing.T) {
 		cfg := &Config{}
 		ConfigureOAuthFromEnv("/redirect/here")(cfg)
 
-		assert.Equal(t, oauth2.Config{}, cfg.oauth)
+		assert.Equal(t, oauth2.Config{}, cfg.oauth.GetConfig())
```

Comment:
This crashes bec `oauth` in an unpopulated field in `cfg` and we're trying to call a function on something that doesn't exist.